rust/sealevel/environments/testnet4/igp/eclipsetestnet/default/igp-accounts.json

```diff
@@ -0,0 +1,5 @@
+{
+  "salt": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "igp_account": "9SQVtTNsbipdMzumhzi6X8GwojiSMwBfqAhS7FgyTcqy",
+  "overhead_igp_account": ""
```

Comment:
nit: for completeness probably worth putting the account in here: `hBHAApi5ZoeCYHqDdCKkCzVKmBdwywdT3hMqe327eZB`